common/src/test/java/io/netty/util/AsciiStringCharacterTest.java

```diff
@@ -233,4 +235,12 @@ public void testBooleanUtilityMethods() {
         assertTrue(new AsciiString(new byte[] { 5 }).parseBoolean());
         assertTrue(new AsciiString(new byte[] { 2, 0 }).parseBoolean());
     }
+
+    @Test
+    public void testEqualsIgnoreCase() {
+        assertThat(AsciiString.contentEqualsIgnoreCase(null, null), is(true));
+        assertThat(AsciiString.contentEqualsIgnoreCase(null, "foo"), is(false));
+        assertThat(AsciiString.contentEqualsIgnoreCase("bar", null), is(false));
+        assertThat(AsciiString.contentEqualsIgnoreCase("FoO", "fOo"), is(true));
+    }
```

Comment:
Just mentioned that this method is in incorrect test class. Can redo, not related.